modules/loader-utils/src/lib/request-utils/request-scheduler.ts

```diff
@@ -25,7 +26,10 @@ const DEFAULT_PROPS: Required<RequestSchedulerProps> = {
   // Specifies if the request scheduler should throttle incoming requests, mainly for comparative testing
   throttleRequests: true,
   // The maximum number of simultaneous active requests. Un-throttled requests do not observe this limit.
-  maxRequests: 6
+  maxRequests: 6,
+  // Specifies a debounce time, in milliseconds. All requests are queued, until no new requests have
+  // been added to the queue for this amount of time.
+  debounce: 0
```

Comment:
Nit: I find it quite useful to specify the time unit in the prop name 
```suggestion
  debounceMs: 0
```

Reply:
Agreed!